ftw/referencewidget/behaviors.py

```diff
@@ -0,0 +1,78 @@
+from ftw.referencewidget.sources import ReferenceObjSourceBinder
+from ftw.referencewidget.widget import ReferenceWidgetFactory
+from plone.app.dexterity import MessageFactory as _
```

Comment:
In 2.0.18 (Pinned by Plone 4.3.11) it's not there.

https://github.com/plone/plone.app.dexterity/blob/2.0.18/plone/app/dexterity/__init__.py